src/pages/api/auth/keyp-sign-in.page.ts

```diff
@@ -0,0 +1,35 @@
+import { sudoPublicDbClient } from 'lib/api/supabase/clients'
+import { NextApiRequest, NextApiResponse } from 'next'
+import { buildJwt, getOrCreateUser } from './utils'
+
+/**
+ * Sign in a keyp wallet without requiring a signature.
+ *
+ * Only POST requests are supported.
+ */
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
@wraeth-eth new api endpoint for signing in a wallet without a challenge message